docker-compose.yaml

```diff
@@ -5,34 +5,67 @@ services:
     hostname: orodha-gateway
     container_name: orodha-gateway
     image: nginx:latest
+    restart: on-failure
     volumes:
       - ./config/gateway/conf.d:/etc/nginx/conf.d
     ports:
       - "80:80"
     networks:
       - orodha-network
 
-  orodha-user-server:
-    hostname: user-service-server
-    container_name: user-service
-    image: edisonstuart/orodha-user-service:latest
+  postgres:
+    image: postgres:13.2
+    restart: unless-stopped
+    hostname: keycloak-db
+    container_name: keycloak-db
+    environment:
+      POSTGRES_DB: ${POSTGRESQL_DB}
+      POSTGRES_USER: ${POSTGRESQL_USER}
+      POSTGRES_PASSWORD: ${POSTGRESQL_PASS}
```

Comment:
@Edison-Stuart Let's have these env vars match exactly, unless there's a specific need for them to be different. Having slight variations in naming increases the risk of misconfiguration (e.g. `POSTGRESQL_PASS` vs `POSTGRES_PASSWORD`)